src/braket/pennylane_plugin/braket_device.py

```diff
@@ -355,6 +366,33 @@ def apply(
 
         return circuit
 
+    def _parametrized_evolution_to_pulse_sequence(
+        self, op: ParametrizedEvolution, dev_wires
+    ) -> PulseSequence:
+        pulse = op.H.pulses[0]  # Assume only one `HardwarePulse`
+        amplitude = partial(pulse.amplitude, op.parameters[0])
+
+        frames = [self._device.frames[f"q{w}_drive"] for w in dev_wires]
```

Comment:
```suggestion
        frames = [self._device.frames[f"q{w}_drive"] for w in pulse.wires]
```
We only want to apply the pulse to the wires included in the `HardwarePulse`

Reply:
In the case of a `ParametrizedEvolution` with a single `HardwarePulse`, is it possible for the wires in the pulse to be different than the wires in the parametrized evolution?

Reply:
Or maybe a handful, but likely some subset